src/components/Rockets.js

```diff
@@ -1,8 +1,44 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchRockets, addReservation, deleteReservation } from '../redux/rockets/rocketsSlice';
+import '../styles/Rockets.css';
 
 function Rockets() {
+  const dispatch = useDispatch();
+  const { rockets } = useSelector((store) => store.rockets);
+
+  const handleAdd = (id) => {
+    dispatch(addReservation(id));
+  };
+
+  const handleDelete = (id) => {
+    dispatch(deleteReservation(id));
+  };
+
+  useEffect(() => {
+    dispatch(fetchRockets());
```

Comment:
- [ ] Kindly make sure that the `Rockets` View/Component is rendered, as for now the action of the rockets gets dispatched every time when changing the view

![Screenshot_20230421_050505](https://user-images.githubusercontent.com/25227022/233546096-d5a5179e-39fa-4195-b078-ddccfb49597d.png)